tests/func/test_import.py

```diff
@@ -305,18 +305,28 @@ def test_cache_type_is_properly_overridden(tmp_dir, scm, dvc, erepo_dir):
     assert scm.is_ignored("foo_imported")
 
 
-def test_pull_imported_directory_stage(tmp_dir, dvc, erepo_dir):
+@pytest.mark.parametrize("dirpath", ["dir", "dir/", "dir/./other"])
```

Comment:
Let's only test the first and second scenario. I don't want to test every scenario that normpath normalizes to.

That'll also simplify the test assertions.

Reply:
Done in 70a959a.

Reply:
@skshetry up to you, but the case you removed was the one happening on the bug report